tests/integration/test_prometheus_protocols/test.py

```diff
@@ -177,3 +177,24 @@ def test_external_tables():
         "DATA mydata TAGS mytags METRICS mymetrics"
     )
     compare_queries()
+
+
+def test_read_auth():
+    node.query("CREATE TABLE prometheus ENGINE=TimeSeries")
+
+    def get(path):
+        headers = {
+            "Content-Type": "application/x-protobuf",
+            "Content-Encoding": "snappy",
+        }
+        return requests.request(
+            url=f"http://{node.ip_address}:{cluster.prometheus_remote_read_handler_port}{path}",
+            method="GET",
+            headers=headers,
+        )
+
+    auth_ok = get("/read_auth_ok")
+    assert auth_ok.status_code == 500
+    assert "DB::Exception: snappy uncomress failed" in auth_ok.text
```

Comment:
What is the purpose of these two lines?

Reply:
Prometheus read handler except proper payload with snappy compression, but there is none. This is just to make sure that 500 error is not related to authorization.

Reply:
It's quite strange to see that we check in this test that `read_auth_ok` returns `snappy uncomress failed`.
We could use the `expose_metrics` protocol instead of `remote_read` in this test however the `expose_metrics` protocol doesn't perform any authentication at all. So I suggest removing these two lines as a quick solution.

Reply:
Removing both lines will simply does not check anything. How about `FIXME` comment instead?

Reply:
ok